cmd/boltzcli/commands.go

```diff
@@ -1879,6 +1896,47 @@ func removeWallet(ctx *cli.Context) error {
 	return nil
 }
 
+func walletSend(ctx *cli.Context) error {
+	client := getClient(ctx)
+	walletId, err := getWalletId(ctx, ctx.Args().First())
+	if err != nil {
+		return err
+	}
+	address := ctx.Args().Get(1)
+	amount := ctx.Args().Get(2)
+
+	request := &boltzrpc.WalletSendRequest{
+		Id:      *walletId,
+		Address: address,
+		Amount:  parseUint64(amount, "amount"),
+	}
+
+	if satPerVbyte := ctx.Float64("sat-per-vbyte"); satPerVbyte != 0 {
+		request.SatPerVbyte = &satPerVbyte
+	}
+
+	txId, err := client.WalletSend(request)
+	if err != nil {
+		return err
+	}
+	fmt.Println("Transaction ID:", txId)
```

Comment:
JSON would be more appropriate to allow for scripting

Reply:
or just the raw output